src/OpenAIGym.jl

```diff
@@ -92,7 +56,8 @@ end
 
 # --------------------------------------------------------------
 
-render(env::AbstractGymEnv, args...) = env.pyenv[:render]()
+render(env::AbstractGymEnv, args...; kwargs...) =
+    pycall(env.pyenv[:render], PyAny; kwargs...)
```

Comment:
I want to write this in `env.pyenv[:render](kwargs...)`,
 I guess there are no difference between them?